ZenScript/src/main/java/stanhebben/zenscript/type/ZenTypeNative.java

```diff
@@ -253,87 +253,87 @@ public void complete(ITypeRegistry types) {
                 }
             }
         }
-        
+
         this.iteratorType = iterator;
         this.iteratorAnnotation = _iteratorAnnotation;
         this.classPkg = _classPkg;
         this.className = _className;
     }
 
     private void checkGetter(Method method, Class cls) {
-        if (method.getReturnType().equals(Void.TYPE)){
+        if (method.getReturnType().equals(Void.TYPE)) {
             throw new RuntimeException("ZenGetter needs a non Void returntype - " + cls.getName() + "." + method.getName());
         }
-        if (method.getParameterCount() > 0){
+        if (method.getParameterCount() > 0) {
             throw new RuntimeException("ZenGetter may not have any parameters - " + cls.getName() + "." + method.getName());
         }
     }
 
     private void checkSetter(Method method, Class cls) {
-        if (method.getParameterCount() != 1){
+        if (method.getParameterCount() != 1) {
             throw new RuntimeException("ZenSetter must have exactly one parameter - " + cls.getName() + "." + method.getName());
         }
         if (!method.getReturnType().equals(Void.TYPE)) {
             throw new RuntimeException("ZenSetter must have a void return type");
         }
     }
-    
+
     public Class getNativeClass() {
         return cls;
     }
-    
-    public void complete(IEnvironmentGlobal environment) {
-        if(iteratorAnnotation instanceof IterableSimple) {
+
+    public void completeIterators(IEnvironmentGlobal environment) {
```

Comment:
Renamed to make it's function more clear
This class wasn't loaded -> Reasons for Iterators not working correctly (i.e., at all)